presentation/src/main/java/org/sopt/at/presentation/home/component/HorizontalBannerPager.kt

```diff
@@ -33,15 +28,15 @@ import org.sopt.at.presentation.R
 fun HorizontalBannerPager(
     @DrawableRes imageList: List<Int>,
     modifier: Modifier = Modifier
-){
+) {
     val coroutineScope = rememberCoroutineScope()
     val pagerState = rememberPagerState(
         initialPage = 4,
         pageCount = { 100 }
     )
 
-    LaunchedEffect(true){
-        while(true){
+    LaunchedEffect(true) {
+        while (true) {
             delay(3000)
             coroutineScope.launch {
                 pagerState.animateScrollToPage(pagerState.currentPage + 1)
```

Comment:
`LaunchedEffect` 는 Compose API레벨에서 `CorutineScope`를 자동으로 관리해주기 때문에,
별도로 `coroutineScope` 를 사용하지 않아도 `suspend` 함수를 실행할 수 있을 것 같습니다 ..!!

![image](https://github.com/user-attachments/assets/bc7a9ac3-ce6b-4ddc-b3f2-1eb54a9e6c98)


```suggestion
    LaunchedEffect(true) {
        while (true) {
            delay(3000)
            pagerState.animateScrollToPage(pagerState.currentPage + 1)
```

반박 의견 환영합니다 ..!! :)

Reply:
그렇군요... 저도 참고해 가겠습니다 ㅎ,,ㅎ!